app/src/main/java/org/matrix/matrixandroidsdk/services/EventStreamService.java

```diff
@@ -264,14 +266,20 @@ private void startWithNotification() {
         mState = StreamAction.START;
     }
 
-    private Notification buildMessageNotification(String from, String body, String roomId) {
+    private Notification buildMessageNotification(String from, String body, String roomId, String roomName) {
         NotificationCompat.Builder builder = new NotificationCompat.Builder(this);
         builder.setWhen(System.currentTimeMillis());
         builder.setContentTitle(from);
+        String name = null;
+        if("" == roomName || null == roomName) {
+            name = "";
+        } else {
+            name = " (" + roomName + "): ";
+        }
         builder.setContentText(body);
         builder.setAutoCancel(true);
         builder.setSmallIcon(R.drawable.ic_menu_small_matrix);
-        builder.setTicker(from + ":" + body);
+        builder.setTicker(from + name + body);
```

Comment:
If ``name`` is not set, we're losing the ``:`` which was here before, as ``:275`` sets it to ``""``.